mobile/src/components/course/components/my-mentor/my-mentor.tsx

```diff
@@ -55,7 +56,9 @@ const MyMentor: FC = () => {
         <ChooseMentor
           course={course}
           mentors={mentors}
+          mentor={mentor}
           isLoading={areMentorsLoading}
+          onChangeMentor={handleMentorCardShownToggle}
```

Comment:
I think we should separate ChooseMentor and mentor card

Reply:
But we already have two components: MentorCard and ChooseMentor.
Maybe I don't understand you correctly ?)